OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+To add/modify user to be part of docker group
+```
+   $ sudo /sbin/usermod -a -G docker <userid>
+```
+
+## 3.2. Set the Proxy if required:
+
+Set up the proxy for docker to connect to the outside world - to access external registries and build a Docker image, set up environment variables for proxy server something like this:
+```
+   export http_proxy=http://www-yourcompany.com:80 
+   export https_proxy=http://www-yourcompany.com:80 
+   export HTTP_PROXY=http://www-yourcompany.com:80 
+   export HTTPS_PROXY=http://www-yourcompany.com:80 
+   export NO_PROXY=localhost,.yourcompany.com 
+```
+
+## 3.3. Create a User Defined network
+
+In this configuration creation of a user defined network will enable communication between the containers just using container names. User defined network option was preferred over the container linking option as the latter is now deprecated. For this setup we will use a user defined network using bridge driver.Create a user defined network using the bridge driver by executing the following command:
+
+        $ docker network create -d bridge <some name>
+
+Sample command ...
+
+        $ docker network create -d WCContentNetwork
+
+## 3.4. Mount a host directory as a data volume
+You need to mount volumes, which are directories stored outside a container's file system, to store WebLogic domain files and any other configuration files. The default location of the volume in the container is `/var/lib/docker/volumes`. 
+
+This option lets you mount a directory from your host to a container as volume. This volume is used to mainly store WebLogic server domain files. The volume is created at this location `/scratch/wccdocker/volumes/wccontentvolume`.
+
+To mount a host directory as a data volume, execute the below command.
+
+> The userid can be anything but it must belong to uid:guid as 1000:1000, which is same as 'oracle' user running in the container.
+
+> This ensures 'oracle' user has access to shared volume.
+
+```
+sudo mkdir -p /scratch/wccdocker/volumes/wccontent
+sudo chown 1000:1000 /scratch/wccdocker/volumes/wccontent
+```
+
+All container operations are performed as **'oracle'** user.
+
+**Note**: If a user already exists with **'-u 1000 -g 1000'** then use the same user. Or modify any existing user to have uid-gid as **'-u 1000 -g 1000'**
+
+## 3.5. Database
+You need to have a running database container or a database running on any machine. 
+The database connection details are required for creating WebCenter Content specific RCU schemas while configuring WebCenter Content domain. 
+
+The Oracle Database image can be pulled from the [Docker Store](https://store.docker.com/images/oracle-database-enterprise-edition) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
```

Comment:
Please remove all mention of the Docker Store. That image is no longer maintained by Oracle. Also, please replace "GitHub Location" and rewrite the end of the sentence as "or build your own [using the Oracle Database Dockerfiles and scripts](https://github.com/oracle/docker-images/tree/master/OracleDatabase/) in this repo.